code/core/arm9/source/GbaColorCorrection/ColorLut.cpp

```diff
@@ -0,0 +1,116 @@
+#include "common.h"
+#include "ColorLut.h"
+#include "GammaLut.h"
+
+// Based on https://gist.github.com/profi200/bfa7be60b3eecb8c43f59000f626c743
+
+u16 gColorLut[COLOR_LUT_SIZE] __attribute__((section(".lutram")));
+
+const ColorProfile* gCurrentPreset = &Agb001; // Default is AGB_001
```

Comment:
Let's not have this state and instead pass it to the `clut_generateColorLut` and `applyColorCorrection` functions directly.